lib/ffmpegCommandService.js

```diff
@@ -401,8 +402,12 @@ class FfmpegCommandService {
         // @TODO: add handling for escaping special characters in bash.  Know it's at least ! that needs escaping.
 
         const text = textOverlay.text.replace(bashCharsToEscapeRegex, "\\$&");
-        const x = this._calculateHorizontalPosition(textOverlay.xLoc);
-        const y = this._calculateVerticalPosition(textOverlay.yLoc);
+        console.log(this._fontConfig);
+
+        const defaultMargin = this._fontConfig.defaultMargin || DEFAULT_FONT_MARGIN;
+
+        const x = this._calculateHorizontalPosition(textOverlay.xLoc, defaultMargin);
```

Comment:
Would it make sense to add a text overlay parameter called xLocMargin or something like that?  Same for yLocMargin.